### PR TITLE
Support parsing multiline step inputs

### PR DESCRIPTION
**Pull request by adborbas**
### Description

Support parsing multiline step inputs by adding `ParseMultilineInput` function to the `input` package.

Function extracted from: https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/config.go#L95

### Changes

- New ParseMultilineInput function
- Switched to go mod
- Removed unnecessary _install-test-tools workflow

### CODE REVIEW

File: input/input.go [hunk 1]
```diff
@@ -73,3 +74,33 @@ func SecureInput(input string) string {
...
+
+// ParseMultilineInput ...
+func ParseMultilineInput(listInput string) []string {
+	trimmedInput := strings.TrimSpace(listInput)
+	if len(trimmedInput) == 0 {
+		return nil
+	}
+
+	separatedItems := []string{trimmedInput}
+	for _, separator := range []string{"\n", `\n`, "|"} {
```
Comment (lpusok) [inline input/input.go:86]: I think it would be better to use a single seperator here and remove "|" or make it optional.

File: input/input.go [hunk 2]
```diff
@@ -73,3 +74,33 @@ func SecureInput(input string) string {
 	}
 	return ""
 }
+
+// ParseMultilineInput ...
+func ParseMultilineInput(listInput string) []string {
```
Comment (lpusok) [inline input/input.go:79]: Please do not use ... as a comment.

**Review by lpusok — CHANGES_REQUESTED** [review]

File: input/input.go (hunk 1 above)
Comment (adborbas) [reply input/input.go:86]: Ok I'll make "|" optional than to allow the original usage at https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/config.go#L95 to be kept as well.

Comment (lpusok) [reply input/input.go:86]: Is "\n", `\n`both needed?

Comment (adborbas) [reply input/input.go]: I'm not sure if both is needed, there is no info on this in google-play-deploy. Do you think we should keep both or should we keep only one?

Comment (lpusok) [reply input/input.go]: The idea may have been to support all line endings on all platforms (\r\n and \n), unsure if it is the right way to do this.

Comment (adborbas) [reply input/input.go]: I would say lets keep this as is. This has been working in google-play-deploy, at least there have been no issues reported on the appList input.

**Review by lpusok — APPROVED** [review]
Looks good. @godrei wdyt?

**Comment by godrei** [comment]
@adborbas why do not we add this functionality to the `stepconf` package: https://github.com/bitrise-io/go-steputils/blob/master/stepconf/example_test.go#L11-L41

I could imagine something like this:

```
type Configs struct {
       ...
	AppPath           []string          `env:"app_path,required,sep[\n,|]"`
       ...
}

```

**Comment by adborbas** [comment]
> @adborbas why do not we add this functionality to the `stepconf` package: https://github.com/bitrise-io/go-steputils/blob/master/stepconf/example_test.go#L11-L41
> 
> I could imagine something like this:
> 
> ```
> type Configs struct {
>        ...
> 	AppPath           []string          `env:"app_path,required,sep[\n,|]"`
>        ...
> }
> ```

That is a much better idea! I will close this PR and open a new one per your suggestion.

**Pull request closed without merging**